lib/app/components/overlay_menu/overlay_menu.dart

```diff
@@ -26,7 +30,12 @@ class OverlayMenu extends HookWidget {
 
     final hideMenu = useCallback(
       () {
-        animationController.reverse().whenComplete(overlayPortalController.hide);
+        animationController.reverse().whenComplete(() {
+          overlayPortalController.hide();
+          if (onClose != null) {
```

Comment:
no need to check if `onClose != null` to make `onClose?.call()`

Reply:
done